javascript/js-forms_handle-form-submit/js/index.js

```diff
@@ -1,3 +1,18 @@
 console.clear();
 
 const form = document.querySelector('[data-js="form"]');
+
+form.addEventListener("submit", (event) => {
+  const formData = new FormData(event.target);
+  const data = Object.fromEntries(formData);
+
+  console.log(data);
+
+  event.preventDefault();
+
+  const firstName = document.getElementById("first-name");
```

Comment:
Instead of creating a new const you could also use "event.target.elements.firstName.focus();"

Reply:
how would I refer to `firstName` if I don't create the variable before?

Reply:
If I understood correctly, when using **event.target.elements.firstName.focus();** by default it uses the name of the HTML-element (so in this case **firstName**), which was already given.